drupal-org.make

```diff
@@ -43,9 +43,7 @@ projects[empty_fields][version] = 1
 projects[entity][type] = module
 projects[entity][version] = 1
 projects[entity_browser][type] = module
-projects[entity_browser][version] = 2.2
-projects[entity_browser][patch][] = https://www.drupal.org/files/issues/2019-07-04/2858438_17.patch
-projects[entity_browser][patch][] = https://www.drupal.org/files/issues/2019-05-21/entity-browser-iframe-scrollbar-3055358-4.patch
+projects[entity_browser][version] = 2.4
 projects[entity_browser][patch][] = https://www.drupal.org/files/issues/2019-10-18/entity-browser-reorder-remove-2851580-67-stable.patch
```

Comment:
Still the old patch here

Reply:
there is no new patch yet, the patch in composer.json is just to be able to test stuff